spec/models/post_spec.rb

```diff
@@ -15,4 +15,44 @@
 
     expect(DiscourseStaffAlias::UsersPostsLink.exists?(post_id: post.id)).to eq(false)
   end
+
+  context "when post_edited" do
```

Comment:
I kind of feel like these tests should be under a different file because we're testing the behaviour of `PostRevisor` here and not really the `Post` model.